semantic-source/semantic-source.cabal

```diff
@@ -15,11 +15,12 @@ category:            Data
 build-type:          Simple
 stability:           alpha
 extra-source-files:
+  CHANGELOG.md
   README.md
 
-
 tested-with:
   GHC == 8.6.5
+  GHC == 8.8.1
```

Comment:
I’m testing this locally until such time as the rest of the project can use 8.8.1.

Reply:
<img width="564" alt="Screen Shot 2019-10-03 at 11 38 14 AM" src="https://user-images.githubusercontent.com/59671/66141799-51d02880-e5d2-11e9-958c-38a49328b2b7.png">